hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp

```diff
@@ -142,10 +142,18 @@
       <h2>Orphan Regions on RegionServer</h2>
     </div>
   </div>
+      <p>
+        <span>
+          The below are Regions we've lost account of. To be safe, run bulk load of any data found in these Region orphan directories back into the HBase cluster.
+          First make sure hbase:meta is in healthy state; run 'hbkc2 fixMeta' to be sure. Once this is done, per Region below, run a bulk
+          load -- '$ hbase completebulkload REGION_DIR_PATH TABLE_NAME' -- and then delete the desiccated directory content (HFiles are removed upon successful load; all that is left are empty directories
```

Comment:
Yuck. In a table with lots of busted regions, this `completeBulkLoad` would be tedious. Maybe we need a new `hbck2 bulkloadOrphanedRegions` command that can identify the orphans, bulk load them, and clean up the husks.